docs/ashell.md

```diff
@@ -14,6 +14,10 @@ arbitrary JavaScript directly from the USB port of a Zephyr enabled device.
 Although the shell would look like a common Unix Bash, it is heavily limited
 due the constraints of the micro-controller.
 
+NOTE: Because the filesystem module is used by ashell, the SPI module is also
```

Comment:
Shouldn't it be easy to add a case to throw an exception in JS and print something if pins 10-13 are set to GPIO when running in ashell?  I think that would be much better than just documenting it here.

Reply:
Sorry for the slow reply.  It would require adding A101 specific code to GPIO, but yes it wouldn't be difficult.  @grgustaf is that something we'd want to add?

Reply:
Yeah, stuff like that will probably eventually come out of the board API work so that it could be more general.